.changeset/fluffy-eggs-film.md

```diff
@@ -0,0 +1,10 @@
+---
+"wrangler": minor
+---
+
+feat(wrangler): allow overriding the unenv preset paths
+
+By default wrangler uses the unenv preset installed via package.json.
+
+Setting `WRANGLER_UNENV_RESOLVE_PATHS` allow to use a local version of the preset.
```

Comment:
I find this naming a bit confusing. I think that we are only using this path to resolve from imports like `unenv/foo/bar` to some absolute path, right? It doesn't affect other imports? Are we really just providing custom paths to the unenv package? Is there some other use for thes paths? If not, then I would expect this to be more like `WRANGLER_UNENV_PACKAGE_PATH`.